--- tools/m2sh/Makefile.old	2010-10-16 15:25:45.000000000 -0300
+++ tools/m2sh/Makefile	2010-11-11 20:42:53.000000000 -0200
@@ -1,7 +1,7 @@
 CFLAGS=-DNDEBUG -DNO_LINENOS -g -I../../src -Isrc -Wall $(OPTFLAGS)
 LIBS=-lzmq -lsqlite3 ../../build/libm2.a $(OPTLIBS)
 
-PREFIX?=/usr/local
+prefix=${prefix:-/usr/local}
 SOURCES=$(wildcard src/*.c)
 OBJECTS=$(patsubst %.c,%.o,${SOURCES})
 TEST_SRC=$(wildcard tests/*.c)
@@ -15,7 +15,7 @@
 dev: all
 
 install: build/m2sh
-	install build/m2sh ${PREFIX}/bin
+	install build/m2sh ${prefix}/bin
 
 build/libm2sh.a: ${LIB_OBJ}
 	mkdir -p build
